### PR TITLE
Updated README.md with latest changes

### PR DESCRIPTION
**Pull request by quic-amitraj**
1. Updated README.md with the latest changes 
2. Updated validated models list and introduction page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,12 @@
...
 *Latest news* :fire: <br>
 
-- [coming soon] Support for more popular [models](#models-coming-soon) and inference optimization techniques like continuous batching and speculative decoding <br>
-* [06/2024] Added support for [GPT-J-6B](https://huggingface.co/EleutherAI/gpt-j-6b)
+- [coming soon] Support for more popular [models](coming_soon_models) and inference optimization technique speculative decoding <br>
+- [08/2024] Added Support for inference optimization technique ```continuous batching```
+- [08/2024] Added support for [Jais-adapted-70b](https://huggingface.co/inceptionai/jais-adapted-70b)
+- [08/2024] Added support for [Jais-adapted-13b-chat](https://huggingface.co/inceptionai/jais-adapted-13b-chat)
+- [08/2024] Added support for [Jais-adapted-7b](https://huggingface.co/inceptionai/jais-adapted-7b)
+- [06/2024] Added support for [GPT-J-6B](https://huggingface.co/EleutherAI/gpt-j-6b)
```
Comment (ochougul) [inline README.md:15]: Who tested these?

**Review by ochougul — CHANGES_REQUESTED** [review]
I don't see a test for Jais models.

**Comment by quic-amitraj** [comment]
Yes tested by @quic-shagun .  All are llama-based architecture only so there is no need to add tests for them.

File: README.md (hunk 1 above)
Comment (quic-amitraj) [reply README.md:15]: Yes tested by @quic-shagun . All are llama-based architecture only so there is no need to add tests for them.

Comment (quic-shagun) [reply README.md:15]: @ochougul 
The jais-adapted-* models are based on Llama architecture developed by inceptionAI 
These models don't need any changes and can run out of the box 
I have verified all three for both non CB and CB

**Review by ochougul — APPROVED** [review]

**Pull request merged**